meteor/server/migration/upgrades/showStyleBase.ts

```diff
@@ -15,39 +19,86 @@ import { evalBlueprint } from '../../api/blueprints/cache'
 import { logger } from '../../logging'
 import { CommonContext } from './context'
 import type { AnyBulkWriteOperation } from 'mongodb'
+import { FixUpBlueprintConfigContext } from '@sofie-automation/corelib/dist/fixUpBlueprintConfig/context'
+import { Blueprint } from '@sofie-automation/corelib/dist/dataModel/Blueprint'
+import { BlueprintFixUpConfigMessage } from '../../../lib/api/migration'
 
-export async function validateConfigForShowStyleBase(
+export async function fixupConfigForShowStyleBase(
```

Comment:
Why is the studio config fixup step ran from the worker whereas the showstyle step is ran from meteor?

Reply:
Following the pattern I started with the other config methods.

I am not 100% sold on it anymore, but my original justification was that we have the job-worker which handles the studio, and doing it there will avoid needing to load the studio blueprints in meteor.  
But a showstyle can be used by multiple studios, so can't be pushed to the job-worker currently.